15.0 Ways to record payments (#2372)
odoo/documentation: content/applications/finance/accounting/receivables/customer_payments/recording.rst

Different ways to record a payment
==================================

In Odoo, payments can either be linked automatically to an invoice or a bill or be stand-alone
records for use at a later date.

If a payment is **linked to an invoice or a bill**, it reduces the amount due of the invoice. You
can have multiple payments related to the same invoice.

If a payment is **not linked to an invoice or a bill**, the customer has an outstanding credit with
your company, or your company has an outstanding debit with a vendor. You can use those outstanding
amounts to reduce unpaid invoices/bills.

Registering payment from an invoice or a bill
=============================================

When you click on :guilabel:`Register payment` on a customer invoice or a vendor bill, it generates

> a new journal entry, reducing the amount due. The counterpart is reflected in an outstanding

Reviewer: Still open

   - If your main bank account is put as :doc:`oustanding account
     <../../bank/setup/outstanding_accounts>`, and the payment is made in Odoo (not related to a
     bank statement), invoices and bills are directly registered in the status :guilabel:`Paid`.
   - If you unreconcile a payment, it still appears in your books but is no longer linked to the
     invoice.
   - If you (un)reconcile a payment in a different currency, a journal entry is created to post the
     Currency Exchange Loss/Gain (reversal) amount.
   - If you (un)reconcile a payment and an invoice having cash basis taxes, a journal entry is
     created to post the Cash Basis Tax (reversal) amount.

.. seealso::
   - :doc:`../../bank/reconciliation/use_cases`

Registering payments not tied to an invoice or a bill
=====================================================

When a new payment is registered via the menu :menuselection:`Customers or Vendors --> Payments`, it
is not directly linked to an invoice or a bill. The account receivable or the account payable have
to be matched manually  with the related invoice or bill.

Matching invoices and bills with payments
-----------------------------------------

> When validating a new invoice or a bill, a blue banner appears when there is an outstanding payment

Reviewer: ```suggestion
When validating a new invoice or bill, a blue banner appears when there is an outstanding payment
```
Tbh it doesn't seem that clear to me... I know what you are referring to but perhaps explain the condition to trigger that banner (that it only appears when Odoo detects a correlating amount and partner)

for this customer or vendor. It can easily be matched from the invoice or the bill, clicking on
:guilabel:`ADD` under :guilabel:`Outstanding Credits` or :guilabel:`Outstanding Debits`.

.. image:: recording/add-option.png
   :align: center
   :alt: Shows the ADD option to reconcile an invoice or a bill with a payment

The invoice or bill is now marked as :guilabel:`In payment` until it is reconciled with the bank
statement.

.. seealso::
   - :doc:`../../bank/reconciliation/use_cases`

Batch payment
-------------

Batch payments allow you to group different payments to ease reconciliation. They are also useful

> when depositing checks to the bank or for Sepa Payments.

Reviewer: ```suggestion
when depositing checks to the bank or for SEPA payments.
```

Go to :menuselection:`Accounting --> Customers --> Batch Payments` or :menuselection:`Accounting -->
Vendors --> Batch Payments`. In the :guilabel:`list view` of payments, you can select several
payments and group them in a batch clicking on Action > Create Batch Payment.

.. seealso::
  - :doc:`../../receivables/customer_payments/batch`
  - :doc:`../../receivables/customer_payments/batch_sdd`

Payments matching
-----------------

> The :guilabel:`Payments matching` tool opens all unreconciled customers invoices or vendors bills

Reviewer: ```suggestion
The :guilabel:`Payments matching` tool opens all unreconciled customer invoices or vendor bills
```
OR customers' invoices or vendors' bills

and gives you the opportunity to process them all one by one, doing the matching of all their

> payments and invoices at once. You can reach it from the :menuselection:`Accounting Dashboard -->

Reviewer: ```suggestion
payments and invoices at once.
```
A bit unclear to me

> Customer Invoices or Vendor Bills`, and click on the ⋮ and select :guilabel:`Payments Matching`. Or

Reviewer: payment matching, or by

going to :menuselection:`Accounting --> Reconciliation`.

.. note::
   During the reconciliation, if the sum of the debits and credits does not match, it means there is

>    still a remaining balance that either needs to be reconciled at a later date or needs to be

Reviewer: ```suggestion 
a remaining balance that either needs to be reconciled at a later date or needs to be
```

   written off directly.

Batch payments matching
-----------------------

To reconcile several outstanding payments or invoices at once, for a specific customer or vendor,
the batch reconciliation feature can be used. Go to :menuselection:`Accounting --> Reporting -->
Aged Receivable or Aged Payable`. You now see all transactions that have not been reconciled yet,

> and when you select a customer or vendor, the :guilabel:`Reconcile` option displays.

Reviewer: ```suggestion
and when you select a customer or vendor, the :guilabel:`Reconcile` option is displayed.
```

.. image:: recording/reconcile-option.png
   :align: center
   :alt: See the reconcile option

Reconciling payments with bank statements
=========================================

> Once a payment has been registered, the status of the invoice or the bill is :guilabel:`In payment`.

Reviewer: ```suggestion
Once a payment has been registered, the status of the invoice or bill is :guilabel:`In payment`.
```

Reviewer: I think you forgot this one

The next step is to reconcile it with the related bank statement line to have the transaction

> finalized and the invoice or the bill marked as :guilabel:`paid`.

Reviewer: ```suggestion
finalized and the invoice or bill marked as :guilabel:`Paid`.
```
